src/stan_math_backend/Expression_gen.ml

```diff
@@ -203,8 +203,19 @@ and gen_distribution_app f =
 (* assumes everything well formed from parser checks *)
 and gen_fun_app ppf f es =
```

Comment:
Also, does this file also include all of Bob's work on expression gen?

Reply:
Yep, that work is the direct predecessor of this file.